SPECS/xcp-ng-release-linstor.spec

```diff
@@ -19,8 +21,12 @@ sed -i "s/\# \(global_filter\)[[:space:]]*=.*/\1 = [ \"r|^\/dev\/drbd.*|\" ]/g"
 
 %files
 %config(noreplace) %{_sysconfdir}/yum.repos.d/xcp-ng-linstor.repo
+%config(noreplace) %{_sysconfdir}/modprobe.d/drbd.conf
 
 %changelog
+* Mon Mar 13 2023 Ronan Abhamon <ronan.abhamon@vates.fr> - 1.2-1
+- Add drbd.conf to prevent idle TCP connections from freezing.
```

Comment:
Maybe mention that it's a modprobe configuration, here and in the commit message too.